### ISSUE TITLE
Pre-Columbian is NOT 100% Forest

### ISSUE THREAD

**Issue by SRGDamia1** [WikiWatershed/model-my-watershed#1264]
The computed Pre-Columbian land-use currently being labeled as 100% forest in the compare screen is *NOT* 100% forest at all.  The code is still using a "Pre-Columbian" condition that converts all "non-natural" land uses into forest, but leaves the others intact.  @aufdenkampe and the rest of us as Stroud completely failed to understand this when we asked the label to be changed to 100% forest.

We will need to figure out at Stroud how to fix this.

Here's a shot I took in an area that is all natural (in this case wetland) to show and obvious difference with 100% forest: 
![capture](https://cloud.githubusercontent.com/assets/5374350/14931613/d3e5d5ea-0e3a-11e6-9730-8366fb0c941a.PNG)

**Comment by SRGDamia1** [comment]
Looking more closely, I think to actually make this into the 100% forest it's labeled as would just involve adjusting the "make_precolumbian" function in tablelookup.py to change all land types instead of just non-natural ones.

Once I hear from the smarter scientists at Stroud, I'll update.

**Comment by ajrobbins** [comment]
@SRGDamia1 any updates on this?

**Comment by SRGDamia1** [comment]
No.  I'm sorry.  It's still a problem, but I haven't looked into it in a long time.
